### PR TITLE
Document webserver deployment for #929

### PR DESCRIPTION
**Pull request by gavinband**
Here's a first go at this - please use / edit as needed.

### CODE REVIEW

File: docs/workflow/deploy.md [hunk 1]
```diff
@@ -12,7 +12,19 @@ reviewed: 18/10/20
 
 \lineskip
 
-Deploying the website is trivial on GitHub, Gitlab and services like Netlify.
+Deploying the website is trivial on an existing webserver, via GitHub or Gitlab, or on services like Netlify.
+
+## Deploying on an existing webserver
+
+The contents of the `__site__` folder can simply be deployed to a path on an existing server as follows.
```
Comment (tlienart) [inline docs/workflow/deploy.md:19]: `__site`

File: docs/workflow/deploy.md [hunk 2]
```diff
@@ -12,7 +12,19 @@ reviewed: 18/10/20
...
+Supposing you wish your site to appear at `http://my.example.com/path/to/my/franklin/site/`.  You would take the following steps:
+
+* Prepare the `__site__` directory by running `optimize( prepath='/path/to/my/franklin/site/', minify = false )`.
```
Comment (tlienart) [inline docs/workflow/deploy.md:22]: idem

File: docs/workflow/deploy.md [hunk 3]
```diff
@@ -12,7 +12,19 @@ reviewed: 18/10/20
...
+Franklin.jl does not use relative links, so this step is needed to ensure that the links between site elements are correct. (The `minify = false` argument is optional but is currently recommended.)
+
+* Copy the contents of the `__site__` directory to the target location using your chosen method.
```
Comment (tlienart) [inline docs/workflow/deploy.md:25]: maybe here after `method` you could add `(e.g. using rsync)`

**Review by tlienart — COMMENTED** [review]
Looks good to me just the `__site__` to change for `__site` but otherwise good, thank you!

File: docs/workflow/deploy.md (hunk 1 above)
Comment (gavinband) [reply docs/workflow/deploy.md]: Fixed in eba69862f708290d218cc1642011fd0e6e5569c6

File: docs/workflow/deploy.md (hunk 3 above)
Comment (gavinband) [reply docs/workflow/deploy.md]: Fixed in eba69862

**Comment by tlienart** [comment]
Great, thanks!

**Pull request merged**